sample/src/main/java/com/canhub/cropper/sample/SMainActivity.kt

```diff
@@ -42,13 +48,26 @@ internal class SMainActivity : AppCompatActivity() {
             hideButtons(binding)
             supportFragmentManager
                 .beginTransaction()
+                .setTransition(FragmentTransaction.TRANSIT_FRAGMENT_OPEN)
                 .replace(R.id.container, SCameraFragmentJava.newInstance())
                 .commit()
         }
+
+        onBackPressedDispatcher.addCallback {
+            if (binding.container.visibility == View.VISIBLE) {
+                binding.image.visibility = View.VISIBLE
+                binding.buttons.visibility = View.VISIBLE
+                binding.container.visibility = View.GONE
+                binding.container.removeAllViews()
+            } else {
+                finish()
+            }
+        }
     }
 
     private fun hideButtons(binding: ActivityMainBinding) {
         binding.image.visibility = View.GONE
         binding.buttons.visibility = View.GONE
+        binding.container.visibility = View.VISIBLE
```

Comment:
Since we are here, let's improve the name of this method, was already wrong before your addition anyway.

Reply:
The structure used in this activity is not good, We have to rewrite a lot of codes anyway. I will do that in the future.